pyaerocom/io/read_eea_aqerep.py

```diff
@@ -435,9 +471,15 @@ def read(self, vars_to_retrieve=None,
         elif isinstance(vars_to_retrieve, str):
             vars_to_retrieve = [vars_to_retrieve]
 
+        if len(vars_to_retrieve) > 1:
+            raise NotImplementedError('So far, only one variable can be read '
+                                      'at a time...')
+        var_name = vars_to_retrieve[0]
+        const.print_log.info('Reading EEA data')
```

Comment:
what's the reason not to use logger directly? (just curious)

Reply:
I want it to crash for now if more variables are input. This is just because of time pressure, it should also be able to read multiple variables in one go.